Changelog.md

```diff
@@ -2,6 +2,68 @@
 
 ### libsdformat 9.X.X (202X-XX-XX)
 
+### libsdformat 9.8.0 (2022-04-26)
+
+1. Polyline geometry DOM
+    * [Pull request #1000](https://github.com/ignitionrobotics/sdformat/pull/1000)
+
+1. Added <shininess> to <material>
+    * [Pull request #985](https://github.com/ignitionrobotics/sdformat/pull/985)
+
+1. Backport ``ign sdf --inertial-stats``
+    * [Pull request #958](https://github.com/ignitionrobotics/sdformat/pull/958)
+
+1. Add L16 pixel format to Camera pixel format conversion function
+    * [Pull request #487](https://github.com/ignitionrobotics/sdformat/pull/487)
+
+1. Anti-aliasing element for <camera><image>
+    * [Pull request #909](https://github.com/ignitionrobotics/sdformat/pull/909)
+
+1. Fix loading nested include with custom attributes
+    * [Pull request #789](https://github.com/ignitionrobotics/sdformat/pull/789)
+
+1. add enable_orientation to 1.6 spec
+    * [Pull request #686](https://github.com/ignitionrobotics/sdformat/pull/686)
+
+1. Include nested model test
+    * [Pull request #Include nested model test](https://github.com/ignitionrobotics/sdformat/pull/Include nested model test)
+
+1. Use SemanticPose API in test
+    * [Pull request #Use SemanticPose API in test](https://github.com/ignitionrobotics/sdformat/pull/Use SemanticPose API in test)
+
+1. Fix test file path in fixed_joint_reduction.cc
```

Comment:
Oops, these 3 lines look bad because the commits were added to #738 without a PR number. We can just remove them from the changelog.